pkg/catalog/diagram.go

```diff
@@ -279,48 +306,6 @@ func (p *Generator) GenerateDataModel(app *sysl.Application) string {
 	return p.CreateFile(plantumlString, plantuml, appName, "types"+p.Ext)
 }
 
-// CreateQueryParamDataModel returns a Query Parameter data model filename.
-func (p *Generator) CreateQueryParamDataModel(CurrentAppName string, param *sysl.Endpoint_RestParams_QueryParam) string {
```

Comment:
??

Reply:
Bungled rebase I think, fixed up in latest commits